account_billing/models/account_billing.py

```diff
@@ -48,13 +48,15 @@ class AccountBilling(models.Model):
     due_date = fields.Date(
         readonly=True,
         states={'draft': [('readonly', False)]},
+        string='Due Date Threshold',
         default=lambda self: fields.Date.context_today(self),
         required=True,
         track_visibility='always',
     )
-    invoice_ids = fields.One2many(
-        comodel_name='account.invoice',
-        inverse_name='billing_id',
+    invoice_ids = fields.Many2many(
```

Comment:
please write name attribute in many2many
```
fields.Many2many(
comodel_name=<object object>,
relation=<object object>,
column1=<object object>,
column2=<object object>,
string=<object object>,
...)
```
https://www.odoo.com/documentation/12.0/reference/orm.html#odoo.fields.Many2many